cg/constants/constants.py

```diff
@@ -139,6 +139,10 @@ class Workflow(StrEnum):
     TAXPROFILER: str = "taxprofiler"
     TOMTE: str = "tomte"
 
+    @classmethod
+    def get_nf_workflows(cls) -> set:
+        return {cls.RNAFUSION, cls.TAXPROFILER, cls.RAREDISEASE, cls.TOMTE}
```

Comment:
Should we use the constant that Annick defined? I would copy that constant and add it here would be an easy conflict to resolve

Reply:
I think this can wait. We will have to update all pytests anyways. So we can address it in a different PR to avoid growing changes here

Reply:
👍